examples/MvcWeb/TheraLang/Services/ResourceService.cs

```diff
@@ -0,0 +1,51 @@
+﻿using System;
+using System.Threading.Tasks;
+using MvcWeb.TheraLang.Entities;
+using MvcWeb.TheraLang.UnitOfWork;
+
+namespace MvcWeb.TheraLang.Services
+{
+    public class ResourceService : IResourceService
+    {
+        private readonly IUnitOfWork unitOfWork;
+
+        public ResourceService(IUnitOfWork unitOfWork)
+        {
+            this.unitOfWork = unitOfWork;
+        }
+
+        public async Task<bool> AddResource(Resource resource)
+        {
+            try
+            {
+                resource.CreatedDateUTC = DateTime.UtcNow;
+                await unitOfWork.Repository<Resource>().Add(resource);
+                await unitOfWork.SaveChangesAsync();
+                
+                return true;
+            }
+            catch
+            {
+                return false;
+            }
+        }
+
+        public async Task<bool> UpdateResource(Resource resource, int updatetById)
+        {
+            try
+            {
+                resource.UpdatedDateUTC = DateTime.UtcNow;
+                resource.UpdatedbyId = updatetById;
+
+                unitOfWork.Repository<Resource>().Update(resource);
+                await unitOfWork.SaveChangesAsync();
+
+                return true;
+            }
+            catch
+            {
+                return false;
+            }
```

Comment:
The same - should be something like:
![image](https://user-images.githubusercontent.com/8735059/68191514-63816300-ffb8-11e9-9310-d00fb6de0e9b.png)